aa_modules/aamod_firstlevel_contrasts.m

```diff
@@ -288,10 +294,22 @@
                 cons{ccount}(inds) = 0;
                 
                 % DIAGNOSTIC
+                
+                % contrast name
                 aas_log(aap,false,contrasts.con(conind).name)
-                for cindex  =  1:max(size(convec_names))
-                    aas_log(aap,false,sprintf('\t%s: %d', convec_names{cindex}, convec(SPM.xX.iC(cindex))))
-                end
+                
+                % list of its event weights
+
+                % if there are a large number of contrasts (e.g., item analysis), echoing
+                % each contrast vector to the command window can take quite awhile...
+                % -- let the user opt-out
+
+                if ~isempty(aas_getsetting(aap,'diagnostics.display_convecs')) && aas_getsetting(aap,'diagnostics.display_convecs')
```

Comment:
You should also add `diagnostics.display_convecs` to the XML.

Reply:
Dang it - I forgot to include the xml in the PR. I will fix..